sdks/go/pkg/beam/core/runtime/xlangx/expansionx/process.go

```diff
@@ -60,11 +63,24 @@ func (e *ExpansionServiceRunner) String() string {
 }
 
 // Endpoint returns the formatted endpoint the ExpansionServiceRunner is set to start the expansion
-// service on. 
+// service on.
 func (e *ExpansionServiceRunner) Endpoint() string {
 	return "localhost:" + e.servicePort
 }
 
+func (e *ExpansionServiceRunner) pingEndpoint(timeout time.Duration) error {
```

Comment:
Notionally this will mean things take a bit longer since we're connecting twice (here, and actual use), but It's probably fine until we determine we don't need to.